src/k8s.ts

```diff
@@ -8,8 +9,9 @@ console.log(JSON.stringify(kc.contexts, null, 2));
 // const k8sApi = kc.makeApiClient(k8s.CoreV1Api);
 const k8sJobApi = kc.makeApiClient(k8s.BatchV1Api);
 
-export async function createK8sJob(projectLink: string, namespace: string): Promise<k8s.V1Job | undefined> {
-  const jobName = 'myjob';
+export async function createK8sJob(projectName: string, projectLink: string, namespace: string): Promise<k8s.V1Job | undefined> {
```

Comment:
As we've defined [ProjectInfo](https://github.com/Eileen-Yu/k8s/blob/cd74aee6a5e6615de45e46bc3167e026566e2184/src/router.ts#L42),
we can replace the arguments here.